database/schemas.py

```diff
@@ -1,19 +1,27 @@
 from typing import List
+from datetime import datetime, timezone
 
-from pydantic import BaseModel, Field
+from pydantic import BaseModel, Field, validator
 
 
 class Detection(BaseModel):
     id: int
     mac: str
     rssi: int
-    baton_uptime_ms: int = Field(alias='uptime_ms')
-    battery_percentage: float = Field(alias='battery')
-    detection_time: int = Field(alias='detection_timestamp')
+    baton_uptime_ms: int = Field(alias="uptime_ms")
+    battery_percentage: float = Field(alias="battery")
+    detection_time: float = Field(alias="detection_timestamp")
```

Comment:
dit was een int, als we daar nu een float van maken, gaat dat mss zaken breken omwille van de gelimiteerde precisie in een float

Reply:
ah hm, in python zijn de floats eigenlijk doubles

Reply:
In java is de conversie tussen timestamps en datetimes met milliseconden. De extra precisie hier maakt dus wel sense denk ik.

Reply:
Zeker omdat we veel detecties binnenkrijgen en het me nuttig lijkt toch het verschil te kunnen zien tussen de meerdere detecties per seconde